public/index.html

```diff
@@ -6,8 +6,169 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Untitled</title>
         <link rel="author" href="humans.txt">
+        <link> <link rel="stylesheet" type="text/css" href="facesmashmain.css">
+    
+    <title>"Facesmash"</title>          
+    
     </head>
+    
     <body>
-        <h1>Audacious Aretes</h1>
+        <ul class="fixedheader clearfix">
+            <li><h1><a href = "/">Facesmash</a></h1></li>
+                <li>
+                <ul>
+                    <li><a href ="/">About Me</a></li>
+                    <li><a href ="/">Sign Up</a></li>
+                    <li><a href ="/">Login</a></li>
+                </ul>
+            </li>
+        </ul>
+        </ul>
+        <form>
+        
+            <div id="main">
+            
+            <p> Connect with your friends and all of the other billions of people in the world that you don't even know. </p>
+
+            <p> Share anything you want. </p>
+
+            <p> Find out things about people you might not want to know about them. </p> 
+
+            <p> It's GREAT! </p>
+           
+            </div>
+            
+
+            <div id = "sideform">
+            
+            <h2> Sign up <h2>
+            
+            <p> It's not free and will never be </p>
+            
+            <input type="text" name="first name" size="15" placeholder="First Name"> 
+            <input type="text" name="last name" size="15" placeholder="Last Name"> 
+            <input type="text"  name="number" size="32" placeholder="Email or Phone Number"> 
+            <input type="text"  name="number" size="32" placeholder="Re-enter Email or Phone Number"> 
```

Comment:
What's a possible collision you're going to have here? Think about line 50 and 51 and how there could be a potential problem when you send this data off.